manual/src/cmds/ocamldoc.etex

```diff
@@ -779,6 +779,7 @@ inline-text: {{inline-text-element}}
         in \ref{sss:ocamldoc-target-specific-syntax}) \\
 @||@&@ '{!' string '}' @ & insert a cross-reference to an element
         (see section \ref{sss:ocamldoc-crossref} for the syntax of cross-references).\\
+@||@&@ '{{!' string '}' inline-text '}' @ & put a cross-reference on the given text. \\
```

Comment:
```suggestion
@||@&@ '{{!' string '}' inline-text '}' @ & insert a cross-reference, overriding the text used. \\
```
or:
```suggestion
@||@&@ '{{!' string '}' inline-text '}' @ & insert a cross-reference with the given text. \\
```

Reply:
Thank you for your suggestion! I prefer the latter.